### PR TITLE
Add Rootstock Hacktivator Program Landing Page

### PR DESCRIPTION
**Pull request by owans**
## What
- Added Rootstock Hacktivator Program Landing Page

## Why
- Introducing Rootstock Hacktivator Program

## Refs
- [Task](https://rsklabs.atlassian.net/browse/DV-424?atlOrigin=eyJpIjoiYTdiMjE4OTI0Yjg3NDU1ZGJlMzk3ZDY3MDYzMGRlMGUiLCJwIjoiaiJ9)

### CODE REVIEW

File: docs/04-resources/02-contribute/hacktivator-program.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+
+> For a comprehensive overview of the guidelines, evaluation, and reward system for content contributions, including detailed criteria for depth, completeness, technical accuracy, educational value, and innovation, please refer to the [Educational Content Evaluation Criteria & Guidelines](https://docs.google.com/document/d/1vA3QK8ZNv5Fgegb0Jv2f03IVIREePDi4lrFE1vsfK7c/edit) and the terms and conditions of the program  [LINK]. These criteria outline how contributions, ranging from technical documentation, written or video guides or tutorials will be assessed and rewarded, ensuring a balanced and fair approach to valuing content quality and impact. This document covers everything from word count appropriateness and inclusion of code examples to technical correctness and relevance to the Rootstock ecosystem, with rewards adjusted based on the value and complexity of each contribution.
```
Comment (ivegabr) [inline docs/04-resources/02-contribute/hacktivator-program.md]: link is missing here -> [LINK]

File: docs/04-resources/02-contribute/hacktivator-program.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+  * The type of contribution and relevant details
+  * A brief description of your work.
+  * A link to your submission (e.g., PR, GitHub for code, or written/video blogs/tutorial links).
+3. Review: Your submission will be reviewed by experts. You may be asked to make revisions or provide more information.
+
+This program is your chance to make a meaningful impact on Rootstock while earning rewards. Ready to contribute? Submit your work and be a part of the Rootstock developer community! 🌍
+
+:::success[T&C]
+
+By participating in this program, you acknowledge that you have read, understood, and agree to abide by all aspects of the program and its terms and conditions. If you do not agree with any part or all of the program and/or its terms and conditions, please refrain from joining or submitting contributions, as your participation will be construed as your full acceptance of these terms.
```
Comment (ivegabr) [inline docs/04-resources/02-contribute/hacktivator-program.md]: is there a link to the T&C?

File: docs/04-resources/02-contribute/hacktivator-program.md (hunk 1 above)
Comment (owans) [reply docs/04-resources/02-contribute/hacktivator-program.md]: Yes, the link will be added soon. Same with the T&C

**Review by ivegabr — APPROVED** [review]
ok

**Pull request merged**